src/Struct/FindologicProduct.php

```diff
@@ -1169,4 +1167,21 @@ protected function addThumbnailUrl(MediaThumbnailEntity $media): void
             $this->images[] = new Image($encodedUrl, Image::TYPE_THUMBNAIL);
         }
     }
+
+    /**
+     * Goes through all given thumbnails, and will only add one thumbnail width as thumbnail. This avoids
+     * the adding of thumbnails in all various sizes.
+     */
```

Comment:
"Go through all given thumbnails and only add one thumbnail image. This avoids exporting thumbnails in all various sizes."